libtransmission/file-posix.cc

```diff
@@ -23,6 +23,10 @@
 #include <sys/stat.h>
 #include <unistd.h> /* lseek(), write(), ftruncate(), pread(), pwrite(), pathconf(), etc */
 
+#ifdef HAVE_FLOCK
+#include <sys/file.h> /* flock() */
+#endif
+
 #ifdef HAVE_XFS_XFS_H
 #include <sys/file.h> /* flock() */
```

Comment:
Please delete this line, it's duplicated.

Reply:
@goodpaperman ^

Reply:
I've done it in https://github.com/transmission/transmission/pull/6908, since it's unlikely that goodpaperman is going to do it (he used his "main" branch for this PR).

Reply:
I've done it in https://github.com/transmission/transmission/pull/6908, since it's unlikely that goodpaperman is going to do it (he used his "main" branch for this PR).